src/main/resources/features/theme 1/05a - tester does.feature

```diff
@@ -55,10 +55,12 @@ Feature: 05a - Tester does...
     And I check the vehicle summary section of the test summary has "Result" of "FAIL"
     And I check the brake test summary section has "Brake results overall" of "Fail"
     And I check the defect section has "Dangerous failures" with value "None recorded"
-    And I check the major failures section of the test summary has "Parking brake efficiency below requirements"
+    And I check the major failures section of the test summary has "None recorded"
     And I check the defect section has "Minors" with value "None recorded"
     And I check the defect section has "PRS" with value "None recorded"
     And I check the defect section has "Advisory text" with value "None recorded"
+    And I check the dangerous failures section of the brake test summary of the test summary has "None recorded"
+    And I check the major failures section of the brake test summary of the test summary has "Service brake efficiency below requirements"
```

Comment:
Is this a new section added to the test summary screen for all users?

Reply:
Yes, brake test defects are now displayed under the brake test section. Before they were under the defects section. See BL-9365